packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/deeply-nested-asyncs/actual.js

```diff
@@ -0,0 +1,13 @@
+async function s(x) {
+  let t = async (y) => {
+    let r = async (z) =>  {
+      await z;
+      return this.x;
+    }
+    await r;

```

Comment:
It doesn't matter, since we're just testing the outputted code, not actually running the code, but shouldn't this be something like `await r(10)` instead of `await r`?  `r` is a function, not a promise, so I'm not sure what the behaviour of awaiting it is.